### ISSUE TITLE
Hardcoded VersionApi error message in connect-sfcluster 

### ISSUE THREAD

**Issue by scaleoutsean** [solidfire/PowerShell#69]
* Error string doesn't correspond to `-VersionApi` specified.
* Separately, the hard-coded v7 confuses the user because it appears it defaults to the then-current release (v10.3 in the case of SF PS Tools 1.5.1), so when a one mistypes the password (for example), their first reaction might be to focus on the wrong `-VersionApi` they used

 ```
PS /home/sean> connect-sfcluster 192.168.1.30 -VersionApi 10

cmdlet Connect-SFCluster at command pipeline position 1
Supply values for the following parameters:
(Type !? for Help.)
Credential
User: admin
Password for user admin: **********

connect-sfcluster : Unable to connect to endpoint: https://192.168.1.30/json-rpc/7.0. An error occurred during the request.
At line:1 char:1
+ connect-sfcluster 192.168.1.30 -VersionApi 10
```

Version info:

```
PS /home/sean> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      6.1.1
PSEdition                      Core
GitCommitId                    6.1.1
OS                             Linux 4.15.0-29-generic #31~16.04.1-Ubuntu SMP Wed Jul 18 08:54:04 UTC 2018
Platform                       Unix

PS /home/sean> Get-Module

ModuleType Version    Name                                ExportedCommands
---------- -------    ----                                ----------------
Manifest   6.1.0.0    Microsoft.PowerShell.Management     {Add-Content, Clear-Content, Clear-Item, Clear-ItemProperty...}
Manifest   6.1.0.0    Microsoft.PowerShell.Utility        {Add-Member, Add-Type, Clear-Variable, Compare-Object...}
Script     2.0.0      PSReadLine                          {Get-PSReadLineKeyHandler, Get-PSReadLineOption, Remove-PSReadLineKeyHandler, Set-PSReadLineKeyHandler...}
Manifest   1.5.1.1    SolidFire.Core                      {Add-SFDrive, Add-SFInitiatorToVolumeAccessGroup, Add-SFNode, Add-SFSnmpNetwork...}
```

**Comment by Salmanius** [comment]
Good catch. That is a fair point. I verified this, and I agree it needs editing to be more clear. We will add a fix for this error in the 1.6 release.

**Comment by arjun960** [comment]
@scaleoutsean The issue is fixed  and changes will be available in the next SDK release. So, closing the issue.
